packages/core/src/node-execution-context/workflow-node-context.ts

```diff
@@ -17,8 +17,7 @@ export class LoadWorkflowNodeContext extends NodeExecutionContext implements IWo
 			this.getNodeParameter = ((
 				parameterName: string,
 				itemIndex: number,
-				// eslint-disable-next-line @typescript-eslint/no-explicit-any
-				fallbackValue?: any,
+				fallbackValue?: unknown,
```

Comment:
oops, this is from my review, lets leave it in rather than restart CI? It's a safe change.